Python_Scripts/selectModelsForMain.py

```diff
@@ -0,0 +1,51 @@
+"""Based on the trajectory comparison, create the main study model list."""
+
+import os
+import shutil
+import pandas as pd
+
+from C import (
+    DIR_MODELS_AMICI, DIR_MODELS_AMICI_FINAL,
+    DIR_MODELS_TRAJ_AMICI)
+
+# trajectory comparison folder
+#  for BDF, abs_error = rel_error = 1e-4, abs_tolerance = rel_tolerance = 1e-12
+trajectory_path = os.path.join(
+    DIR_MODELS_TRAJ_AMICI,
+    'comparisons_BDF_1e-12_1e-12/comparisons_1e-04_1e-04')
```

Comment:
I think this is a conceptional problem here:
We don't want to do this comparison for only one setting. For the moment, we can keep this line here. But this needs to be fixed together with #10 ....
Because once #10 is fixed, we want to iterate here over all those settings...

Reply:
True, this affects "what models to include in the study finally" --> bias correction